docs/manifest.json

```diff
@@ -143,11 +143,65 @@
               {
                 "title": "Deleting a CNAME",
                 "path": "/docs/portal/cnames/deleting-a-cname.md"
-              }
-            ]
-          },
-          { "title": "Global Presence", "path": "/docs/portal/global-presence.md" }
-        ]
-      }
-    ]
-  }
+              },
+              {
+                  "title": "Managing Registry Projects and Users",
```

Comment:
Managing registry projects and managing registry users are closely coupled. That's why we've made them 2 sub-menus under "Image Registry" menu on portal. So please combine registry projects and register users here on documentation site, and put them under a same parent menu "Registry Management" which will be at the same level as other menus like "Applications", "CNAMEs", etc.

Reply:
I created the parent topic called "Registry Management." Below it, I have two children topics called ""Managing Registry Projects" and ""Managing Registry Users." This structure resembles the structure of the ECP application, and organizes projects and users within their own subcategories. Please confirm this is okay, or whether you want me to remove the two subtopics and collect all project and user topics within the parent topic of "Registry Management."

Reply:
I am afraid we have to remove the two subtopics. I recall Sergey said that 3-level menu structure is not supported. 
It shall be fine to collect all project and user topics within the parent topic.

Reply:
Done, thank you.